src/wp-includes/class-wp-post-type.php

```diff
@@ -396,6 +396,55 @@ final class WP_Post_Type {
 	 */
 	public $rest_controller;
 
+	/**
+	 * The controller for this post type's revisions REST API endpoints.
+	 *
+	 * Custom controllers must extend WP_REST_Controller.
+	 *
+	 * @since 6.4.0
+	 * @var string|bool $revisions_rest_controller_class
+	 */
+	public $revisions_rest_controller_class;
+
+	/**
+	 * The controller instance for this post type's revisions REST API endpoints.
+	 *
+	 * Lazily computed. Should be accessed using {@see WP_Post_Type::get_revisions_rest_controller()}.
+	 *
+	 * @since 6.4.0
+	 * @var WP_REST_Controller $revisions_rest_controller
+	 */
+	public $revisions_rest_controller;
+
+
+	/**
+	 * The controller for this post type's autosave REST API endpoints.
+	 *
+	 * Custom controllers must extend WP_REST_Controller.
+	 *
+	 * @since 6.4.0
+	 * @var string|bool $autosave_rest_controller_class
+	 */
+	public $autosave_rest_controller_class;
+
+	/**
+	 * The controller instance for this post type's autosave REST API endpoints.
+	 *
+	 * Lazily computed. Should be accessed using {@see WP_Post_Type::get_autosave_rest_controller()}.
+	 *
+	 * @since 6.4.0
+	 * @var WP_REST_Controller $autosave_rest_controller
+	 */
+	public $autosave_rest_controller;
+
+	/**
+	 * A flag to direct the REST API controllers for autosave / revisions should be registered before/after the post type controller.
```

Comment:
Wording is confusing to me, proposed alternative:

```suggestion
	 * A flag to register the post type REST API controller after its associated autosave / revisions controllers, instead of before.
```
Or,

```suggestion
	 * A flag to register the post type REST API controller after its associated autosave / revisions controllers, instead of before. Registration order affects route matching priority.
```

Slight nitpick on the name, as well, since reading this identifier on its own could be interpreted to alter the registration of all routes relating to this controller, as opposed to the controller's main route only. I don't have a suggestion for a better name, though.